planner/core/exhaust_physical_plans.go

```diff
@@ -2822,6 +2822,21 @@ func (lw *LogicalWindow) GetPartitionKeys() []*property.MPPPartitionColumn {
 	return partitionByCols
 }
 
+// Duration vs Datetime is invalid comparison as TiFlash can't handle it so far.
+func (lw *LogicalWindow) checkComparison(frameBound *FrameBound) bool {
```

Comment:
rename it to `checkComparisonForTiFlash`

Reply:
> checkComparisonForTiFlash

okk